src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  if(days<3){
+    return days*RENT_COST;
```

Comment:
The comparison `days < 3` is correct, but the literal `3` should be replaced with a constant that explains its meaning.